src/main/java/dev/openfga/sdk/api/client/OpenFgaClient.java

```diff
@@ -277,11 +277,15 @@ public CompletableFuture<ClientWriteResponse> write(ClientWriteRequest request,
         configuration.assertValid();
         String storeId = configuration.getStoreIdChecked();
 
-        if (options != null && !options.disableTransactions()) {
-            return writeTransactions(storeId, request, options);
+        if (options == null) {
+            options = new ClientWriteOptions();
```

Comment:
Why not just leave it as null and then treat it as writeTransactions?

Reply:
(Which breaks a test where I had intentionally written a chunk size of 1)

Reply:
Addressing in 150b77290ea4ef975d9a5f2f98098d1236128b3d